lib/block/header.ex

```diff
@@ -84,9 +104,9 @@ defmodule Block.Header do
       h.receipts_root,
       h.logs_bloom,
       h.difficulty,
-      (if h.number == 0, do: <<>>, else: h.number),
```

Comment:
I don't understand this syntax change.

Reply:
@hayesgm ```if``` is a method that takes boolean expression, and keyword as parameters. so formatter thinks it should look better like ```if(h.number == 0, do: <<>>, else: h.number)```

Reply:
oh cool, though it must be a macro, not a method. In either case, this appears less clear than the previous code.